aspnetcore/migration/inc/remote-app-setup.md

```diff
@@ -35,7 +35,17 @@ SystemWebAdapterConfiguration.AddSystemWebAdapters(this)
     });
 ```
 
-In the options configuration method passed to the `AddRemoteApp` call, you must specify an API key which is used to secure the endpoint so that only trusted callers can make requests to it (this same API key will be provided to the ASP.NET Core app when it is configured). The API key is a string and must be parsable as a GUID (128-bit hex number). Hyphens in the key are optional.
+In the options configuration method passed to the `AddRemoteAppServer` call, you must specify an API key which is used to secure the endpoint so that only trusted callers can make requests to it (this same API key will be provided to the ASP.NET Core app when it is configured). The API key is a string and must be parsable as a GUID (128-bit hex number). Hyphens in the key are optional.
+
+3. Add the `SystemWebAdapterModule` HttpModule to your web.config if it wasn't already added by the [.Net Upgrade Assistant](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.upgradeassistant):
```

Comment:
```suggestion
3. [Optional] Add the `SystemWebAdapterModule` module to your web.config if it wasn't already added by NuGet (such as using SDK style projects for ASP.NET Core).
```

This is not something added by the .NET Upgrade Assistant. Rather, it's something added by NuGet. However, if you are using SDK style project for ASP.NET Framework, this will need to be added manually as NuGet only applies these transforms on old-style project files.